earth_enterprise/src/common/performancelogger.cpp

```diff
@@ -18,19 +18,131 @@
 #include <unistd.h>
 #include <sys/syscall.h>
 #include <time.h>
-
+#include <cstdlib>
 #include "common/performancelogger.h"
 #include "common/notify.h"
 #include "common/timeutils.h"
+#include <cassert>
 
 using namespace std;
 using namespace getime;
 
-// Initialize static members of Profiler class
-PerformanceLogger * const PerformanceLogger::_instance = new PerformanceLogger();
+string ioFileName;
```

Comment:
These look like they can be member variables, since they pertain to what the logger is doing.